proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -60,36 +55,42 @@ message ConnectCluster {
 
 message ConnectorStatus {
   message Connector {
+    // State of the connector instance.
     string state = 1;
     string worker_id = 2;
     string trace = 3;
   }
+  // Name of connector.
   string name = 1;
   Connector connector = 2;
+  // Status of connector tasks. For more information, see the [https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/#connector-tasks](Monitor Connectors) documentation.
   repeated TaskStatus tasks = 3;
   string type = 4;
```

Comment:
Can we add a description for connector.type? Is there documentation we can link out to for more details?

Reply:
This is a connector type, whether it's a "source" or a "sink".
https://docs.confluent.io/platform/current/connect/references/restapi.html#get--connectors

Reply:
```suggestion
  // Connector type, source or sink.
  string type = 4;
```